packages/components/src/Slider/Slider.stories.js

```diff
@@ -109,11 +136,22 @@ storiesOf('Form/Controls/Slider', module).add('default', () => (
 				/>
 			</div>
 			<div style={delimiterStyle}>
-				<p>with range</p>
+				<p>with range (inclusive)</p>
+				<Slider
+					onChange={action('onChange')}
+					min={0}
+					max={100}
+					value={[25, 75]}
+					allowCross={false}
+				/>
+			</div>
+			<div style={delimiterStyle}>
+				<p>with range (exclusive)</p>
 				<Slider
 					onChange={action('onChange')}
 					min={0}
 					max={100}
+					mode={'exclusive'}
```

Comment:
Please use exported constant here instead
```suggestion
					mode={SLIDER_MODE.EXCLUSIVE}
```